bucket/microsoft11.json

```
@@ -0,0 +1,20 @@
+{
+    "description": "The Microsoft Build of OpenJDK is a new no-cost long-term supported distribution and Microsoft's new way to collaborate and contribute to the Java ecosystem.",
+    "homepage": "https://www.microsoft.com/openjdk",
+    "version": "11.0.11-9",
+    "license": {
+        "identifier": "GPL2 with Classpath Exception",
```

```suggestion
        "identifier": "GPL-2.0-only WITH Classpath-exception-2.0",
```
Per https://spdx.org/licenses/GPL-2.0-with-classpath-exception.html